### PR TITLE
[author] [new] Add diva.js@4.0.0

### PR DESCRIPTION
**Pull request by magoni**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Wrong indent in package.json, please use [tools/fixFormat.js](https://github.com/cdnjs/cdnjs/blob/master/tools/fixFormat.js) to help you fix the indent of package.json, and it'll be even better if you can sync/rebase your branch with our latest master branch, thanks!

**Comment by PeterDaveHello** [comment]
@magoni if you don't mind, I can take this.

**Comment by PeterDaveHello** [comment]
@magoni it passed the test now, would you please also add `keywords`, `license` and [`autoupdate`](https://github.com/cdnjs/cdnjs#enabling-gitrecommended-or-npm-auto-update) in package.json? Thanks!

**Comment by magoni** [comment]
Revised!

File: ajax/libs/diva.js/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "version": "4.0.0",
+  "description": "Diva.js is a JavaScript document image viewer designed to present multi-page documents at high resolutions.",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/DDMAL/diva.js.git",
+    "basePath": "/build/",
+    "files": [
+      "js/diva.min.js",
+      "css/diva.min.js",
+      "img/*"
```
Comment (PeterDaveHello) [inline ajax/libs/diva.js/package.json]: Should use `"img/**/*"` here to grab files recursively.

Comment (magoni) [inline ajax/libs/diva.js/package.json]: Fixed, thanks for your help!

Comment (PeterDaveHello) [inline ajax/libs/diva.js/package.json]: np

**Comment by PeterDaveHello** [comment]
hmmmmm ... I think the directory structure is wrong, should be the same as the build folder, I'll fix it now.

**Comment by PeterDaveHello** [comment]
fixed in 3b6eff5

**Pull request merged**